.github/configs/feature.yaml

```diff
@@ -12,12 +12,9 @@ develop:
   solc: 0.8.21
 eip7692:
   evm-type: eip7692
-  fill-params: --fork=CancunEIP7692 ./tests/osaka
-  solc: 0.8.21
-eip7692-osaka:
```

Comment:
@danceratopz I'm not sure if removing this is ok? this release build failed in the CI on 7702 tests.

I'm not sure what was the purpose of these two releases being split, but maybe if it was just filling for Osaka (not the now-obsolete `CancunEIP7692`), then I guess we can at this point get rid of?

Reply:
Yes, I believe `CancunEIP7692` is an artifact from before EOF was scoped for Osaka. And, yes, it's better not exclude Prague tests for an EOF release :-) (including `./tests/prague` is probably also a historical leftover).

From our side, this can be removed. Perhaps ask Danno or @chfast if you're in any doubt.

Reply:
Asked. It's OK